### PR TITLE
styled the calc page

### PR DESCRIPTION
**Pull request by muzahidOly**

### CODE REVIEW

File: housinghelper/listings/forms.py [hunk 1]
```diff
@@ -22,7 +22,20 @@ class TestForm(forms.Form):
     loan_total = forms.IntegerField()
     interest_rate = forms.FloatField()
     loan_term = forms.IntegerField()
+    
     amount = forms.DecimalField(
-            widget= calculation.FormulaInput('loan_total*(((interest_rate)*(1+interest_rate)^loan_term)/(1+interest_rate)^(loan_term - 1))') # <- using single math expression
+        
+
+            widget= calculation.FormulaInput(
+                'loan_total*(((interest_rate/12)*(1+(interest_rate/12))**loan_term)/((1+(interest_rate/12))**loan_term-1))'
```
Comment (muzahidOly) [inline housinghelper/listings/forms.py:30]: Fixed the calculated total of mortagage

File: housinghelper/listings/forms.py [hunk 2]
```diff
@@ -22,7 +22,20 @@ class TestForm(forms.Form):
...
+                
+                ) # <- using single math expression
      #widget=calculation.FormulaInput('(0.28*price*term)+(down_payment)')
+
+
     )
+    
+    
+    Total_cost_over_term = forms.DecimalField( widget= calculation.FormulaInput(
```
Comment (muzahidOly) [inline housinghelper/listings/forms.py:39]: Added the total cost for the house after X term

File: housinghelper/listings/templates/listings/calc_form.html [hunk 3]
```diff
@@ -1,13 +1,77 @@
...
+
+
+
+		
 	<div class="px-6 py-4">
+
+		
 	<h1 class="mb-4 text-4xl font-extrabold tracking-tight leading-none text-gray-900 md:text-5xl lg:text-6xl text-black-600 text-center">Home Affordability Calculator </h1>
-	<h1 class="mb-4 text-4xl  tracking-tight leading-none text-gray-900 md:text-3xl lg:text-6xl text-black-500 text-center">Using the 26/36 Rule </h1>
+	<h1 class="mb-4 text-4xl  tracking-tight leading-none text-gray-900 md:text-3xl lg:text-6xl text-black-500 text-center">Using <a href="https://www.mymove.com/mortgage/mortgage-calculation/#:~:text=These%20factors%20include%20the%20total,)%5En%20%E2%80%93%201%5D">MyMove™ </a> </h1>
```
Comment (muzahidOly) [inline housinghelper/listings/templates/listings/calc_form.html:16]: Credited the site

**Comment by sylo7** [comment]
Restyled calculation page is much more usable and visually appealing. Calculation function being redone also gives us much more accurate calculations

**Comment by Afn8** [comment]
Styled calculation form looks great, also great to see it working well.

**Pull request merged**